### ISSUE TITLE
long link over multiple lines enclosed in "<>" does trigger MD034 (no bare URL)

### ISSUE THREAD

**Issue by SvenMW** [DavidAnson/markdownlint#353]
example link, which can be split onto multiple lines afaik:

```
(<https://review.opendev.org/gitweb?p=openstack%2Fnova-specs.git;hb=refs%2F
  changes%2F50%2F310050%2F9;f=specs%2Fbacklog%2Ffs-attach-detach.rst>)
```

still triggers MD034 for me in vs code:
```
{
	"resource": "REDACTED",
	"owner": "markdownlint",
	"code": {
		"value": "MD034",
		"target": {
			"$mid": 1,
			"external": "https://github.com/DavidAnson/markdownlint/blob/v0.22.0/doc/Rules.md#md034",
			"path": "/DavidAnson/markdownlint/blob/v0.22.0/doc/Rules.md",
			"scheme": "https",
			"authority": "github.com",
			"fragment": "md034"
		}
	},
	"severity": 4,
	"message": "MD034/no-bare-urls: Bare URL used",
	"source": "markdownlint",
	"startLineNumber": 64,
	"startColumn": 5,
	"endLineNumber": 64,
	"endColumn": 78
}
```

any help would be appreciated.

**Comment by SvenMW** [comment]
mhm nevermind, it does not seem to be really supported to split lonf links onto multiple lines?

do you know any solution to this,beside using a url shortener?

I found this on stackoverflow, but it doesn't seem to work for my markdownflavor (the link brakes at line break).

**Comment by DavidAnson** [comment]
Link references are a great solution here. I make extensive use in https://github.com/DavidAnson/markdownlint-cli2/blob/main/README.md and group all the links at the end of the file where I disable the line length restriction.

**Comment by SvenMW** [comment]
Thanks for the idea! Will check out if that works for our documents.

**Comment by SvenMW** [comment]
mhm, you are using this switch there to disable the check:

```
<!-- markdownlint-disable line-length -->
```
however this does not seem to work for me in vs code?

When I add this line it still complains about the check.

Any tips would be appreciated.
